tools/gather-sysinfo/gather-sysinfo.go

```diff
@@ -164,6 +164,7 @@ func kniExpectedCloneContent() []string {
 	return []string{
 		// generic information
 		"/proc/cmdline",
+		"/sys/devices/system/cpu/offline",
```

Comment:
I would have put this below right next to "// KNI-specific CPU infos" but not a big deal, we can reorder later.
What the options are to test this change?

Reply:
Done. I don't see any test for must-gather. 
Could create one, extract the tar and check the offline cpus with performanceProfile config or something like this one, no int must gather but valid for validate offlineCPUS
https://github.com/openshift-kni/performance-addon-operators/blob/078eda67f7ceb54ea8923435c52ba7c8563dc365/functests/utils/nodes/nodes.go#L216

Reply:
we don't have tests and we need to fix this. It would be nice to add in this PR but is not blocking. Let's start designing how the test suite could look like and let's file issues(/cards) to track this work.
Then the actual testsuite bootstrap can happen on followup work.